packages/api-key-stamper/src/tink/elliptic_curves.ts

```diff
@@ -136,35 +136,86 @@ function getY(x: bigint, lsb: boolean): bigint {
 }
 
 /**
- * Decodes a public key in _compressed_ format.
+ *
+ * Given x and y coordinates of a JWK, checks whether these are valid points on
+ * the P-256 elliptic curve.
+ *
+ * P-256 only
+ *
+ * @param x x-coordinate
+ * @param y y-coordinate
+ * @return boolean validity
+ */
+function validateUncompressedXY(x: bigint, y: bigint): boolean {
```

Comment:
Something to note here: it's great that you've thought about validating that the point is on the curve, but this is going a bit "above and beyond": when this JWK structure is _actually_ imported with `crypto.importKey`, this validation will occur anyway (`crypto.importKey` will throw an exception)

Reply:
Right yeah makes sense -- wanted to provide similar guarantees within our code for compressed as we did for uncompressed and catch errors within our own code first but I see how it's an error case either way.